content/300-guides/050-database/850-using-prisma-with-planetscale.mdx

```diff
@@ -0,0 +1,240 @@
+---
+title: 'Using Prisma with PlanetScale'
+metaTitle: 'Using Prisma with PlanetScale'
+metaDescription: 'Guide to using Prisma with PlanetScale'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+Prisma and [PlanetScale](https://planetscale.com/) together provide a development arena that optimizes rapid, type-safe development of data access applications, using Prisma's ORM and PlanetScale's highly scalable MySQL-based platform.
+
+This document discusses the concepts behind using Prisma and PlanetScale, explains the commonalities and differences between PlanetScale and other database providers, and leads you through the process for configuring your application to integrate with PlanetScale.
+
+</TopBlock>
+
+## What is PlanetScale?
+
+PlanetScale uses the [Vitess](https://vitess.io/) database clustering system to provide a MySQL-compatible database platform. Features include:
+
+- **Enterprise scalability.** PlanetScale provides a highly available production database cluster that supports scaling across multiple database servers.
```

Comment:
This _could_ have some wording that connection limits do not really exist with PlanetScale, which makes it amazing to use in a serverless context as well as you do not have to worry about that problem any more (I think we have an article where we describe this in the docs and common workaround, so we could link to that for context). This is _really_ what makes PlanetScale so amazing for our users.

Reply:
Done